docs/farming.md

```diff
@@ -27,8 +27,11 @@ The address of your account will be necessary at the last step.
 ## 🖼️ Windows Instructions
 
 <details>
-
-### 📝 Windows Installation
+<summary>
+Details
+</summary>
+<p>
```

Comment:
Not sure what these changes are about, but they seem useless and render incorrectly. And please, no HTML except `<details>` tag.

Reply:
The core issue I was attempting to resolve here actually pertains to the documentation I've been working on. 

As you mentioned on a PR review on that repo, it is not good to have two places to try and keep up with docs, so there is a plugin which will load this readme into the docs. Problem being the <details> tag is not rendering properly on docusarus as it is not defined according to html standard. <details><summary><p> etc.

Reply:
Of course it is defined in a standard and supported by all major browsers:
https://html.spec.whatwg.org/multipage/interactive-elements.html#the-details-element
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details#browser_compatibility

Reply:
I don't think I explained that very clearly. I understand there is a standard for the Details element lol. What I was trying to say is that while GitHub renders the `details` element just fine without the `summary` or a `p` tag. Docusarus does not render the same .md file correctly. (Does not show the details when clicked). So this was an attempted fix to allow it to render properly both on GitHub & Docusarus. 

But in hindsight its probably just better to remove all the HTML(including details element), and just have native markdown which would be a much more straight forward solution, and leave us with a more maintainable documentation moving forward

Reply:
Yeah, let's do that